src/test/java/com/endava/internship/mocking/repository/InMemPaymentRepositoryTest.java

```diff
@@ -0,0 +1,68 @@
+package com.endava.internship.mocking.repository;
```

Comment:
For JUnit 5 is recommended to omit the public modifier for test classes, test methods, and lifecycle methods.
![image](https://user-images.githubusercontent.com/80094766/195038111-85a4ce0f-04ca-4412-917d-8491d170d29b.png)
https://junit.org/junit5/docs/snapshot/user-guide/